app/src/styles/StyleCardView.tsx

```diff
@@ -1,82 +1,79 @@
 import { styled } from '@mui/material/styles';
 import { Box, Typography } from '@mui/material';
 import ButtonLink from '@/components/ButtonLink';
-
-// Color Scheme:
-// #ECF2F8: Light blue
-// #96b6d4: Mild blue
-// #477096: Dark blue
-// #4F4846: Black
-// #FDF8F5: White
+import {LIGHT_BLUE, MILD_BLUE, DARK_BLUE, BLACK, WHITE } from '@/styles/ColorScheme';
 
 // Define custom styles
 const StyledReportDetails = styled(Box)({
     '& p': {
         margin: 0,
         padding: '5px 0',
-        color: '#4F4846',
+        color: `${BLACK}`,
```

Comment:
Is there a reason why this couldn't just be `color: BLACK,`?

Reply:
Since '#4F4846' or `${BLACK}` is a little bit different than normal black. I think it's a shade of black (lighter) but more compatible with our blue scheme, at least I think it is.